README.md

```diff
@@ -105,6 +105,37 @@ func main() {
 }
 ```
 
+### Custom comparision function
+
+```go
+package main
+
+import(
+    "net/http"
+
+    "github.com/goji/httpauth"
+)
+
+func DummyBasicAuth() func(http.Handler) http.Handler {
+    opts := httpauth.AuthOptions{
+        Realm: "Restricted",
+        AuthFunc: func(u,p string) bool { return true },
```

Comment:
I can probably add a more realistic example, but can get quiet complex. I already did that for https://github.com/sensu/uchiwa/pull/244, that's also the reason for this PR.